Gemfile.lock

```diff
@@ -508,9 +506,8 @@ GEM
     zeitwerk (2.4.2)
 
 PLATFORMS
-  ruby
```

Comment:
At some point, maybe we should have a conversation about the PLATFORMS values and whether it's OK for them to keep changing.

Reply:
I never look at this, I just run `bundle install`